commands/util.go

```diff
@@ -12,11 +14,14 @@ import (
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/cli/context/docker"
 	"github.com/docker/cli/cli/context/kubernetes"
+	ctxstore "github.com/docker/cli/cli/context/store"
 	dopts "github.com/docker/cli/opts"
+	apis "github.com/docker/compose-on-kubernetes/api"
```

Comment:
Can we eliminate dependency on docker/compose-on-kubernetes?

Reply:
it is a deps in `github.com/docker/cli/cli/context/kubernetes`.

I will the use clientcmd instead.